SPECS/jitterentropy/makefile.patch

```diff
@@ -0,0 +1,11 @@
+--- ./Makefile	2021-04-06 08:53:18.554954462 -0700
++++ ./Makefile	2021-04-14 09:44:09.424948057 -0700
+@@ -70,7 +70,7 @@
+ cppcheck:
+ 	cppcheck --force -q --enable=performance --enable=warning --enable=portability *.h *.c
+ 
+-install: install-man install-shared install-includes
++install: install-man install-static install-includes
```

Comment:
Instead of using a patch, wouldn't it be easier to just run `make install-man install-static install-includes` straight from the spec? Also, we could then skip generating the man files, since we don't use them and can make the package smaller.

Reply:
Good catch